extensions/wikia/AdEngine/js/spec/lookup/adapter/appnexus.spec.js

```diff
@@ -4,7 +4,8 @@ describe('ext.wikia.adEngine.lookup.adapter.appnexus', function () {
 	function getAppNexus() {
 		return modules['ext.wikia.adEngine.lookup.adapter.appnexus'](
 			mocks.geo,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the declaration of "mocks" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)